site/gatsby-site/src/templates/cite.js

```diff
@@ -1,33 +1,37 @@
 import React, { useEffect, useMemo, useState } from 'react';
-import AiidHelmet from 'components/AiidHelmet';
-import Layout from 'components/Layout';
-import Citation from 'components/cite/Citation';
-import ImageCarousel from 'components/cite/ImageCarousel';
-import BibTex from 'components/BibTex';
-import { getCanonicalUrl } from 'utils/getCanonicalUrl';
+import { Spinner } from 'flowbite-react';
+import AiidHelmet from '../components/AiidHelmet';
+import Layout from '../components/Layout';
+import Citation from '../components/cite/Citation';
+import ImageCarousel from '../components/cite/ImageCarousel';
+import BibTex from '../components/BibTex';
+import { getCanonicalUrl } from '../utils/getCanonicalUrl';
 import { format, isAfter, isEqual } from 'date-fns';
 import { useModal, CustomModal } from '../hooks/useModal';
-import Timeline from 'components/visualizations/Timeline';
-import IncidentStatsCard from 'components/cite/IncidentStatsCard';
-import IncidentCard from 'components/cite/IncidentCard';
-import Taxonomy from 'components/taxa/Taxonomy';
-import { useUserContext } from 'contexts/userContext';
-import SimilarIncidents from 'components/cite/SimilarIncidents';
+import Timeline from '../components/visualizations/Timeline';
```

Comment:
Please undo these changes, it throws away the benefits of using the webpack mappings from:  https://github.com/responsible-ai-collaborative/aiid/blob/master/site/gatsby-site/gatsby-node.js#L71

Reply:
Done